frameworks/Java/firenio/pom.xml

```diff
@@ -14,12 +14,22 @@
 
     <packaging>jar</packaging>
 
+    <repositories>
+        <repository>
+            <id>snapshots</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
     <dependencies>
 
         <dependency>
             <groupId>com.firenio</groupId>
             <artifactId>firenio-all</artifactId>
-            <version>1.3.2</version>
+            <version>1.3.5-SNAPSHOT</version>
```

Comment:
I realize that some frameworks are currently using `SNAPSHOT`, but we're going to start prohibiting this. I was actually unaware that snapshots could be updated at will. Could you update this to a more recent stable version and I will let others know to do the same. Thanks!

Reply:
Agree with you, updated.